File: packages-gen/dart/libs/xt-ui/lib/state/session.dart
project(session, profile, capabilities) {
  return <dynamic, dynamic>{
    "authenticated":true == (session ?? <dynamic, dynamic>{})["authenticated"],
    "user_id":(session ?? <dynamic, dynamic>{})["user_id"],
    "profile":profile ?? <dynamic, dynamic>{},
    "capabilities":capabilities ?? <dynamic, dynamic>{}
  };
}

capablep(state, capability_id) {
  return true == state["capabilities"][capability_id];
}

File: packages-gen/js/libs/xt-ui/state/session.js
function project(session,profile,capabilities){
  return {
    "authenticated":true == (session || {})["authenticated"],
    "user_id":(session || {})["user_id"],
    "profile":profile || {},
    "capabilities":capabilities || {}
  };
}

function capablep(state,capability_id){
  return true == state["capabilities"][capability_id];
}

module.exports = {["project"]:project,["capablep"]:capablep}